internal/evaluation_target/application_snapshot_image/application_snapshot_image_test.go

```diff
@@ -139,43 +139,84 @@ func createSimpleAttestation(statement *in_toto.Statement) oci.Signature {
 	return signature
 }
 
-func TestWriteInputFiles(t *testing.T) {
+const simpleAttestationJSONText = `{
+  "_type": "",
+  "predicateType": "https://slsa.dev/provenance/v0.2",
+  "subject": null,
+  "predicate": {
+    "buildType": "https://tekton.dev/attestations/chains/pipelinerun@v2",
+    "builder": {
+    "id": ""
+    },
+    "invocation": {
+    "configSource": {}
+    }
+  }
+}`
```

Comment:
I'm bike shedding here, but it feels odd to have the two space indent here. Would tabs work?

Reply:
That's a good point. It's JSON so it doesn't matter. Switched to tabs to stay consistent with the rest of the file.